src/V3Randomize.cpp

```diff
@@ -709,7 +709,17 @@ class ConstraintExprVisitor final : public VNVisitor {
     void visit(AstAssocSel* nodep) override {
         if (editFormat(nodep)) return;
         FileLine* const fl = nodep->fileline();
-        if (VN_IS(nodep->bitp(), CvtPackString) && VN_IS(nodep->bitp()->dtypep(), BasicDType)) {
+        if (VN_IS(nodep->bitp(), VarRef)
+            && VN_AS(nodep->bitp(), VarRef)->varp()->dtypep()->basicp()
+            && VN_AS(nodep->bitp(), VarRef)->dtypep()->basicp()->keyword()
+                   == VBasicDTypeKwd::STRING) {
```

Comment:
Note not equivalent below; one had ->varp() and other did not, I think the varp() shouldn't have been needed.
```suggestion
        if (VN_IS(nodep->bitp(), VarRef)
            && VN_AS(nodep->bitp(), VarRef)->isString()) {
```